src/pages/UserEdit.js

```diff
@@ -11,33 +11,35 @@ import { makeStyles } from "@material-ui/core";
 import theme from "../util/theme";
 
 // Project resources
-import { getUser, updateUser } from "../util/firebase/user"
-import { IUser } from "../types";
-import TextField from "../components/TextField";
+import { getUser, updateUser } from "../util/firebase/user";
+import { TextFormField } from "../components/FormFields";
 
 const UserEdit = () => {
   const { t } = useTranslation();
   const { userId } = useParams();
   const [user, setUser] = useState();
   const [chainId, setChainId] = useState();
-  const classes = makeStyles(theme.form)();
+  const classes = makeStyles(theme)();
   const { register, handleSubmit, setValue } = useForm();
   const history = useHistory();
 
   const onSubmit = (user) => {
     updateUser(userId, user);
-    history.push({ pathname:`/chains/${chainId}`, state: { message: t("saved") } });
+    history.push({
+      pathname: `/chains/${chainId}`,
+      state: { message: t("saved") },
+    });
   };
 
   useEffect(() => {
-    (async function() {
+    (async function () {
       try {
         const user = await getUser(userId);
         setUser(user);
         setChainId(user.chainId.id);
-        const fields = ['name', 'address', 'email', 'phoneNumber'];
+        const fields = ["name", "address", "email", "phoneNumber"];
         fields.forEach((field) => setValue(field, user[field]));
-      } catch(error) {
+      } catch (error) {
         console.error(error);
       }
     })();
```

Comment:
This useEffect function is running an infinite loop. Fixed it by adding an empty array as a second argument: 
```
  useEffect(() => {
    someLogicHere();
  }, []);
```

Reply:
Fixed, thanks!